content/en/news/releases/1.15.x/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 1.15.x Releases
+description: Announcements for the 1.15 release and its associated patch releases.
+weight: 15
+list_by_publishdate: true
+layout: release-grid
+decoration: dot
+---
+
+We are pleased to announce the release of Istio 1.15!
+
+{{< relnote >}}
+
+This is the third Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.15.0 published. Special thanks are due to the release managers Sam Naser and Aryan Gupta from Google, Ziyang Xiao from Intel and Daniel Hawton from solo.io. As always, our gratitude goes to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
```

Comment:
I expect solo.io here should be Solo.io or just Solo. I'm not sure how they differentiate whether.io should be added. I do see on their website, `At Solo.io, we have helped thousands of organizations` which means that they typically capitalize it unless it's the URL...

Reply:
@dhawton Thoughts on Solo vs Solo.io?

Reply:
Solo.io is preferred.